ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/forms/useReportFormValues.tsx

```diff
@@ -87,6 +92,16 @@ export const defaultReportFormValues: ReportFormValues = {
     },
 };
 
+export const emailSubjectValidation = yup
+    .string()
+    .required('An email subject is required')
+    .max(maxEmailSubjectLength, `Limit your input to ${maxEmailSubjectLength} characters or less.`);
+
+export const emailBodyValidation = yup
+    .string()
+    .required('An email body is required')
+    .max(maxEmailBodyLength, `Limit your input to ${maxEmailBodyLength} characters or less.`);
```

Comment:
Grammar nit: "fewer" instead of less when talking about countable things.
```suggestion
    .max(maxEmailBodyLength, `Limit your input to ${maxEmailBodyLength} characters or fewer.`);
```